src/context.rs

```diff
@@ -262,6 +262,17 @@ impl ContextImpl {
         unsafe { StructType::new(LLVMStructCreateNamed(self.0, c_string.as_ptr())) }
     }
 
```

Comment:
There is a `LLVMGetTypeByName` function befor 12.0. Should I add support for it too?